.github/workflows/Build.yml

```diff
@@ -88,6 +96,15 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install ninja scikit-ci-addons cmake~=3.18.0
+    - name: Install Monarch self-hosted build dependencies
+      if: matrix.os == 'self-hosted-x64'
+      run: |
+        apt install -y \
```

Comment:
Why apt over apt-get? I see a warning indicating that apt's interface is not stable.

Reply:
Fingers work faster than brain sometimes.  I'll change this to apt-get then merge.